content/project/trademark-sublicense.adoc

```diff
@@ -0,0 +1,105 @@
+
+4.LIMITED WARRANTY; DISCLAIMERS OF WARRANTIES AND LIABILITIES
+
+4.1 SPI warrants that it owns the right to sublicense the Trademark in the United States and anywhere else that SPI owns the Trademark. SPI MAKES NO OTHER WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED WARRANTY OF MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE (E.G., THAT ANY AUTHORIZED GOODS/ SERVICES TO WHICH THE SUBLICENSEE MARK IS APPLIED ARE COMPLIANT WITH ANY STANDARDS) AND ANY WARRANTY WITH RESPECT TO NON-INRINGEMENT OF THE TRADEMARK, THE SUBLICENSEE MARK, OR OF THE AUTHORIZED GOODS/SERVICES BEARING THE TRADEMARK AND/OR SUBLICENSEE MARK.
+
+4.2 EXCEPT FOR ANY BREACH OF THE LIMITED WARRANTY ABOVE, SPI EXPRESSLY DISCLAIMS LIABILITY FOR ANY DAMAGES INCURRED BY SUBLICENSEE AND THIRD PARTY CLAIMS OF ANY KIND THAT MAY ARISE OUT OF SUBLICENSEE'S USE OF THE TRADEMARK AND/OR THE SUBLICENSEE MARK.
+
+4.3 Without these limitations on warranties and disclaimers, and without other limitations on SPI's obligations set forth elsewhere in this Sublicense (including, without limitation, those identified in Section 1.3 (Consent) and Section 1.4 (Non Exclusivity)), SPI would not grant the Sublicense at any royalty rate or under any circumstances. These disclaimers of liability and warranties shall be effective as to any country in which Sublicensee (or another for or on behalf of Sublicensee) manufactures, sells, licenses or performs Authorized Goods/Services using the Sublicensee Mark.
+
+5. Miscellaneous.
+
+5.1 No Sublicensee Rights.
+Sublicense shall not have any rights against SPI.
+
+5.2 Assignment.
+SPI may assign or transfer its rights under this Sublicense to a subsidiary, affiliate or parent of SPI. Sublicensee may not assign or transfer this Sublicense (whether by agreement or operation of law) without the prior written consent of SPI, which consent may be given or denied in its sole discretion. This Sublicense shall in all cases be binding on the parties' successors and assigns.
+
+5.3 Notices.
+All notices provided for in this Sublicense shall be in writing and shall be effective when actually received by the addressees at the addresses listed in the Application Approval Statement. Either party may change its address to which notices or requests shall be directed by written notice to the other party (which may be via email at the last known email address), but until such change of address has been received any notice or request sent to the addresses listed below shall be effective upon mailing or sending (if by email) and shall be considered as having been received.
+
+5.4 Severability.
+If any term or provision of this Sublicense shall be held invalid under any applicable law, rule or regulation of any jurisdiction, then that provision notwithstanding, this Sublicense shall remain in full force and effect and such provision shall be deemed deleted.
+
+5.5 Entire Agreement.
```

Comment:
5.5 has a strikethrough on the wiki page, ascii doc doesn't seem to have that built in https://wiki.jenkins.io/display/JENKINS/Trademark+Sublicense

I've removed 5.5 and moved everything up one, is that correct?  / ideas?